### PR TITLE
Added responsive utlity nav. Tweaked sticky nav JS.

### PR DESCRIPTION
**Pull request by kanastasakis26**

### CODE REVIEW

File: src/ts/mmui-layout/index.ts [hunk 1]
```diff
@@ -29,10 +29,18 @@ export var getLayoutBreakSize = function(width){
...
-        let offset = mainNav.offsetTop;
+        let offset;
+        if(defaultOffset> -1){
+            offset = defaultOffset;
+        }
+        else {
+            offset = mainNav.offsetTop;
+
+        }
+        console.log(offset);
```
Comment (besseym) [inline src/ts/mmui-layout/index.ts]: remove console.log

File: src/sass/mmui-styles/mmui_theme.scss [hunk 2]
```diff
@@ -115,4 +116,41 @@ a.util-nav-link{
...
+  .navbar-toggler{
+    margin-left: 10px;
+  }
+}
+
+.desktop-nav{
+  display: none;
+}
+
+@media only screen and (min-width: 992px) {
```
Comment (besseym) [inline src/sass/mmui-styles/mmui_theme.scss]: Use Bootstrap media query mixin:
`@include media-breakpoint-up(lg) {`

File: src/ts/mmui-layout/index.ts (hunk 1 above)
Comment (kanastasakis26) [reply src/ts/mmui-layout/index.ts]: Removed

File: src/sass/mmui-styles/mmui_theme.scss (hunk 2 above)
Comment (kanastasakis26) [reply src/sass/mmui-styles/mmui_theme.scss]: Changed

File: src/ts/mmui-layout/index.ts [hunk 3]
```diff
@@ -29,10 +29,17 @@ export var getLayoutBreakSize = function(width){
  * setStickyNavBehavior("main-nav")
  *
  */
-export function setStickyNavBehavior(navid){
+export function setStickyNavBehavior(navid, defaultOffset=-1){
```
Comment (besseym) [inline src/ts/mmui-layout/index.ts]: You could probably set a default value for the navid as well
`navid='main-nav'`

**Pull request merged**